### ISSUE TITLE
Sum of OR features?

### ISSUE THREAD

**Issue by JadeIV** [Cairath/tools-not-included#13]
I'd like a way to specify a total number of collected map features to check for.

Example: I'd like to have a map with at least 6 water-generating geysers, but I don't care exactly what sort of geyser (water, p.water, steam, ice) they are. There certainly are seeds that meet that criteria, but at present, there's no precise way to search for them; there are no seeds with 6 or more of any one type of water-generating geyser and setting an OR command with some lower minimum of each type is going to both show some seeds that don't meet the desired requirement and skip some seeds that meet it.

**Comment by nevir** [comment]
Yes please! Even if it's special cased to just sum together all water producing geysers, I think that would be extremely helpful
